nncf/experimental/tensor/numpy_functions.py

```diff
@@ -158,7 +154,49 @@ def _(
     return np.where(condition, x, y)
 
 
-@functions.zeros_like.register(np.ndarray)
-@functions.zeros_like.register(np.number)
+@registry_numpy_types(fns.zeros_like)
 def _(a: Union[np.ndarray, np.number]) -> np.ndarray:
     return np.zeros_like(a)
+
+
+@registry_numpy_types(fns.stack)
+def _(x: Union[np.ndarray, np.number], axis: int = 0) -> List[np.ndarray]:
+    return np.stack(x, axis=axis)
+
+
+@registry_numpy_types(fns.unstack)
+def _(x: Union[np.ndarray, np.number], axis: int = 0) -> List[np.ndarray]:
+    return [np.squeeze(e, axis) for e in np.split(x, x.shape[axis], axis=axis)]
+
+
+@registry_numpy_types(fns.moveaxis)
+def _(a: np.ndarray, source: Union[int, List[int]], destination: Union[int, List[int]]) -> np.ndarray:
+    return np.moveaxis(a, source, destination)
+
+
+@registry_numpy_types(fns.mean)
+def _(a: Union[np.ndarray, np.number], axis: Union[int, List[int]] = None, keepdims: bool = False) -> np.ndarray:
+    return np.mean(a, axis=axis, keepdims=keepdims)
+
+
+@registry_numpy_types(fns.round)
+def _(a: Union[np.ndarray, np.number], decimals: int = 0) -> np.ndarray:
+    return np.round(a, decimals=decimals)
+
+
+@registry_numpy_types(fns.binary_operator)
+def _(
+    a: Union[np.ndarray, np.number], b: Union[np.ndarray, np.number], operator_fn: Callable
+) -> Union[np.ndarray, np.number]:
+    # Run operator with disabled warning
+    with np.errstate(invalid="ignore", divide="ignore"):
```

Comment:
Why exactly do we need warnings disabled during these ops? Don't we want to know about places where we try to divide by zero, and try to handle this at the common code level by ensuring that there are no places where we unexpectedly try to divide by zero?

Reply:
It's need to sync behavior for numpy and torch, torch has not analog of `errstate` to raise error or warnings. 
Default behavior in both backends is that dividing by zero return `inf`, as result in common methods requires addition checks and it's not depend from warnings.
And in NNCF no any places where used `np.errstate(divide="raise"):` only "ignore".